modules/500-upmeter/openapi/config-values.yaml

```diff
@@ -16,6 +16,8 @@ properties:
 
       **CAUTION!** Setting this value to one that differs from the current one (in the existing PVC) will result in disk re-provisioning and data loss.
 
+      **CAUTION!** When migrating Prometheus with the specified storage Class name, the pods will hang in the Pending state and it will be necessary to delete the old PVC and restart the pod manually. The transfer of Prometheus to other nodes when using emptyDir takes place without the need to restart the hearths.
```

Comment:
```suggestion
      **CAUTION!** When migrating Upmeter with the specified storageClass name, the pods will hang in Pending state and it will be necessary to delete the old PVC and manually restart the pod. Migration of Prometheus to other nodes using emptyDir occurs without the need to restart the pods.
```